### ISSUE TITLE
How to set the ABLUnit output path

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#314]
I want to use the "Failed unit tests should be fixed" rule but i don't see how i set the directory of my unit test output. Is there a property or is this some fixed dir?

**Comment by gquerret** [comment]
This metric is not pushed to SQ for now. We'll see if we can do that during summer time.

**Comment by gquerret** [comment]
The right way to do that will be to use `sonar.tests` to specify the path where tests are located, and then either use [Generic test data](https://docs.sonarqube.org/display/SONAR/Generic+Test+Data) (but requires a conversion of ABLUnit report to the expected format), or a new sensor to directly read ABLUnit format (which is based on standard JUnit format).
In the mean time, please note that `sonar.tests` shouldn't be used as the OpenEdge plugin doesn't handle correctly this property.

**Comment by gquerret** [comment]
Using [XmlTask](http://www.oopsconsultancy.com/software/xmltask), you can convert ABLUnit output to the expected format:
```xml
    <xmltask source="test-output/results.xml" dest="test-output/sonar.xml">
      <rename path="//testsuites" to="testExecutions" />
      <attr path="//testExecutions" attr="version" value="1" />
      <rename path="//testsuite" to="file" />
      <rename path="//file/@name" to="path" />
      <rename path="//testcase" to="testCase" />
      <!-- Duration should be converted to milliseconds -->
      <!-- Left as an exercise to the reader... -->
      <remove path="//testCase/@time" />
      <attr path="//testCase" attr="duration" value="0" />
    </xmltask>
```

Then specify `sonar.testExecutionReportPaths=test-output/sonar.xml` during the SQ analysis.

Correct handling of sonar.tests coming soon

**Comment by gquerret** [comment]
Merged in master
